services/web/src/components/RichTextEditor/index.js

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { omit } from 'lodash';
+import PropTypes from 'prop-types';
+import Editor from '@draft-js-plugins/editor';
+import { EditorState, RichUtils, convertToRaw, convertFromRaw } from 'draft-js';
```

Comment:
Did we add these modules to the package?

Reply:
hah, good call

Reply:
done